lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
             try
             {
                 TangFactory.GetTang()
-                    .NewInjector(mapInputPipelineDataConverterConfig)
+                    .NewInjector(mapOutputPipelineDataConverterConfig)
                     .GetInstance<IPipelineDataConverter<TMapOutput>>();
```

Comment:
Do you use the result of the instance of IPipelineDataConverter? What is the purpose of the GetInstance() here?